workers/unity/Assets/Playground/Scripts/Metrics/MetricSendSystem.cs

```diff
@@ -0,0 +1,75 @@
+using System.Collections.Generic;
+using Unity.Entities;
+using UnityEngine;
+using Improbable.Gdk.Core;
+
+namespace Playground
+{
+    public class MetricSendSystem : ComponentSystem
```

Comment:
Not actually sure if this needs to be executed in a specific part of the update loop. Might be worth looking up what deltaTime will be for different systems. I suspect this is fine provided we don't put it in FixedUpdate though.